kazakh/ualeikum.md

```diff
@@ -0,0 +1 @@
+Баяғы бір заманда бай мен кедей көрші өмір сүріпті. Олардың араздығы сондай, бірін-бірі көргісі келмейді екен. [Байдың үйі](bai/bai.md) кең, әдемі болыпты. Іші толған – алтын мен күміс. Ал кедейдің үйі ағаштан жасалған, төбесін бұтамен жапқан қора екен. Алтын мен күміс түгілі, нанын әзер тауып жепті. Егін егіп, оны суарып, күтіп, орып, өз күнін өзі көріпті.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.